src/shapes/object.class.js

```diff
@@ -634,6 +634,13 @@
      */
     absolutePositioned: false,
 
+    /**
+     * Indicates that this is the highlight layer image buffer
+     * @type boolean
+     * @default false
+     */
+    isHighlightLayer: false,
```

Comment:
This confused me at first since I already set this on the image, but I realized I never defined that property as an official one on any object (oh the joys of JS being so loose). So this is good since now this is officially defined for all objects and defaults to false :)